### PR TITLE
xp-pen-g430: rename to pentablet-driver & 20190820 -> 1.2.13.1

### PR DESCRIPTION
**Pull request by IvarWithoutBones**
###### Motivation for this change
Renamed the xp-pen-g430 package to pentablet-driver, as i noticed all drivers in this series are exactly the same. Also updated it from 20190820 -> 1.2.13.1.

I'm copying them to `$out/bin` seperately now as to not have the annoying capitalization of the executable. Note that `config.xml` has to be next to the executable for it to start.

Their versioning is a tad confusing but this came out like 5 months later, so I'm sure it's newer.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/misc/drivers/pentablet-driver/default.nix [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{ stdenv, mkDerivation, fetchzip, autoPatchelfHook, libusb1, libX11, libXtst, qtbase, libglvnd }:
+
+mkDerivation rec {
```
Comment (prusnak) [inline pkgs/misc/drivers/pentablet-driver/default.nix:3]: Replace `mkDerivation` with `stdenv.mkDerivation` and you can drop the `mkDerivation` from the first line

**Review by prusnak — CHANGES_REQUESTED** [review]
Maybe also add the following to the `pkgs/top-level/aliases.nix`:

```
xp-pen-g430 = pentablet-driver; # added 2020-05-03
```

File: pkgs/misc/drivers/pentablet-driver/default.nix (hunk 1 above)
Comment (symphorien) [reply pkgs/misc/drivers/pentablet-driver/default.nix:3]: no I think when qt is involved, using the mkDerivation from args is correct.

Comment (prusnak) [reply pkgs/misc/drivers/pentablet-driver/default.nix:3]: Ah, correct. I missed that part.

Comment (IvarWithoutBones) [reply pkgs/misc/drivers/pentablet-driver/default.nix:3]: Yep, i replaced it with `wrapQtAppsHook` as this is preffered, at least thats what I've been told on [another PR](https://github.com/NixOS/nixpkgs/pull/84117#issuecomment-617983911).

**Comment by IvarWithoutBones** [comment]
> Maybe also add the following to the `pkgs/top-level/aliases.nix`:
> 
> ```
> xp-pen-g430 = pentablet-driver; # added 2020-05-03
> ```

Done :+1:

File: pkgs/top-level/all-packages.nix [hunk 2]
```diff
@@ -26346,7 +26346,9 @@ in
 
   bcompare = libsForQt5.callPackage ../applications/version-management/bcompare {};
 
-  xp-pen-g430 = libsForQt5.callPackage ../misc/drivers/xp-pen-g430 {};
+  pentablet-driver = libsForQt5.callPackage ../misc/drivers/pentablet-driver { };
+
+  xp-pen-g430 = pentablet-driver; # Renamed on 2020-05-03
```
Comment (prusnak) [inline pkgs/top-level/all-packages.nix:26351]: This line belongs to `pkgs/top-level/aliases.nix`, not `pkgs/top-level/all-packages.nix`.

**Review by prusnak — CHANGES_REQUESTED** [review]

File: pkgs/top-level/all-packages.nix (hunk 2 above)
Comment (prusnak) [reply pkgs/top-level/all-packages.nix:26351]: Another option is to add there the following:

```
xp-pen-g430 = throw "xp-pen-g430 was renamed to pentablet-driver"; # added 2020-05-03
```

Comment (IvarWithoutBones) [reply pkgs/top-level/all-packages.nix]: Oh, I'm sorry. Misread your review.. Fixed.

**Review by prusnak — APPROVED** [review]

**Pull request merged**